.pre-commit-config.yaml

```diff
@@ -12,6 +12,10 @@ repos:
     -   id: check-toml
     -   id: check-xml
     -   id: check-yaml
+        args:
+          # https://github.com/pre-commit/pre-commit-hooks/issues/701
+          # Necessary so that non-standard tags do not throw failures.
+          - '--unsafe'
```

Comment:
Reviewing

Reply:
Thanks, @scottschreckengaust ! If we want to reserve the full load of other YAML files in the repo, we can duplicate the hook and make it file-specific, e.g. apply the `--unsafe` arg only on the mkdocs.yml, but keep the rest of the files within the normal hook scope. Let me know if that's preferred and I can make the change, but overall the `--unsafe` flag disables full loading of the YAML to prevent unsafe code execution and focus on linting only.